client/keys/list_test.go

```diff
@@ -16,6 +16,16 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+func kbCleanup(t *testing.T, kb keyring.Keyring, keys ...string) func() {
```

Comment:
Just a suggestion, this name IMO sounds a bit more self-explanatory:

```suggestion
func cleanupKeys(t *testing.T, kb keyring.Keyring, keys ...string) func() {
```